srv/salt/ceph/openstack/cinder-backup/key/default.sls

```diff
@@ -0,0 +1,16 @@
+{% set prefix = pillar['openstack_prefix'] + "-" if 'openstack_prefix' in pillar else "" %}
```

Comment:
Couldn't this be the standard:

`{% set prefix = salt['pillar.get']('openstack_prefix' + "-", "") %}`

Reply:
That would be rather less ugly. Thank you.

Reply:
Actually, that doesn't quite work. `salt['pillar.get']('openstack_prefix' + "-", "")` tries to get a value named 'openstack-prefix-'. I need to get a value named 'openstack-prefix', and append '-' to the value, iff the value is present. Would the following be clearer?

````
{% set prefix = salt['pillar.get']('openstack_prefix', "") %}
{% set prefix = prefix + "-" if prefix else "" %}
````

Or is there a neater one-liner than I'm just not seeing yet?